docs/platform/one-click/deploying-ruby-on-rails-with-one-click-apps/index.md

```diff
@@ -0,0 +1,160 @@
+
+{{< content "deploy-one-click-apps">}}
+
+The [Ruby on Rails Options](#ruby-on-rails-options) section of this guide provides details on all available configuration options for this app.
+
+### Ruby on Rails Options
+
+You can configure your Drupal App by providing values for the following fields:
+
+| **Field** | **Description** |
+|:--------------|:------------|
+| **Rails Application name** | The name for your rails application. *Required*. |
+
+### Linode Options
+
+After providing the app specific options, provide configurations for your Linode server:
+
+| **Configuration** | **Description** |
+|:--------------|:------------|
+| **Select an Image** | Debian 9 is currently the only image supported by Ruby on Rails One-Click Apps, and it is pre-selected on the Linode creation page. *Required*. |
+| **Region** | The region where you would like your Linode to reside. In general, it's best to choose a location that's closest to you. For more information on choosing a DC, review the [How to Choose a Data Center](/docs/platform/how-to-choose-a-data-center) guide. You can also generate [MTR reports](/docs/networking/diagnostics/diagnosing-network-issues-with-mtr/) for a deeper look at the network routes between you and each of our data centers. *Required*. |
+| **Linode Plan** | Your Linode's [hardware resources](/docs/platform/how-to-choose-a-linode-plan/#hardware-resource-definitions).  *Required*. |
+| **Linode Label** | The name for your Linode, which must be unique between all of the Linodes on your account. This name will be how you identify your server in the Cloud Manager’s Dashboard. *Required*. |
+| **Root Password** | The primary administrative password for your Linode instance. This password must be provided when you log in to your Linode via SSH. It must be at least 6 characters long and contain characters from two of the following categories: lowercase and uppercase case letters, numbers, and punctuation characters. Your root password can be used to perform any action on your server, so make it long, complex, and unique. *Required*. |
+
+When you've provided all required Linode Options, click on the **Create** button. **Your Ruby on Rails app will complete installation anywhere between 2-5 minutes after your Linode has finished provisioning**.
+
+## Getting Started after Deployment
+
+### Access Ruby on Rails
+
+After Ruby on Rails has finished installing, you will be able to access Ruby on Rails from the console via ssh with your Linode's IPv4 address:
+
+1.  [SSH into your Linode](/docs/getting-started/#connect-to-your-linode-via-ssh) and [create a limited user account](/docs/security/securing-your-server/#add-a-limited-user-account).
+
+1.  Log out and log back in as your limited user account.
+
+1.  Update your server:
+
+        sudo apt-get update && apt-get upgrade
+
+1.  Ruby comes with some pre-made scripts to get you started. One of these is a blog, to start this, use the following command:
```

Comment:
Split "One of these is a blog. To begin with the blog example,..." into two sentences.

Reply:
updated